mods/aerexplorer.wh.cpp

```diff
@@ -3021,6 +3144,13 @@ BOOL Wh_ModInit(void)
     LOAD_MODULE(ExplorerFrame)
     HOOK_SYMBOLS(ExplorerFrame, explorerframeDllHooks)
 
+    VS_FIXEDFILEINFO *pVerInfo = GetModuleVersionInfo(ExplorerFrame, nullptr);
```

Comment:
Just after merging, I noticed that the check should have been done before `HOOK_SYMBOLS`. Currently, it doesn't help as `Wh_SetFunctionHook` will still be called and fail.